src/backend/marsha/core/xapi.py

```diff
@@ -45,24 +69,19 @@ def get_locale(self):
 
     def get_actor_from_website(self, homepage, user):
         """Return the actor property from a website context"""
-        return {
-            "objectType": "Agent",
-            "account": {
-                "homePage": homepage,
-                "mbox": f"mailto:{user.email}",
-                "name": str(user.id),
-            },
-        }
+        return BaseXapiAgentWithAccount(
+            account=BaseXapiAccount(homePage=homepage, name=str(user.id)),
+            name=user.username,
```

Comment:
`name` field of the actor is optionnal. Should we avoid setting it, facilitating our future work on anonymization?
If we decide to keep it, should'nt it be a full name instead of username?